src/docfx/build/page/BuildPage.cs

```diff
@@ -16,33 +16,24 @@ public static async Task<List<Error>> Build(Context context, Document file)
         {
             Debug.Assert(file.ContentType == ContentType.Page);
 
-            var (errors, pageModel, inputMetadata) = await Load(context, file);
-            var (generateErrors, outputMetadata) = await GenerateOutputMetadata(context, file, inputMetadata);
-            errors.AddRange(generateErrors);
+            var (errors, pageModel) = await Load(context, file);
```

Comment:
Need to clarify the terms used here, shall we call the original JSON `sourceModel`, input to template `pageModel`, and the current model (after JINT/mustache) stored in DHS `templateModel` ?

```suggestion
            var (errors, sourceModel) = await Load(context, file);
```

Reply:
sounds good to me